decidim-budgets/app/commands/decidim/budgets/add_line_item.rb

```diff
@@ -10,7 +10,7 @@ class AddLineItem < Rectify::Command
       # project - The the project to include in the order
       # current_user - The current user logged in
       def initialize(current_order, project, current_user)
-        @order = current_order
+        @order = current_order&.persisted? ? current_order : nil
```

Comment:
if current_order is passed but not persisted it will create a new Order. Is it correct?

Reply:
Yes, the _why_ is explained in an answer to Leo => https://github.com/decidim/decidim/pull/5865#discussion_r395261728

Reply:
@tramuntanal refactored as suggested by @leio10 => https://github.com/decidim/decidim/pull/5865#discussion_r401620195